1.11/administering-clusters/licenses/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: layout.pug
+navigationTitle:  Licenses
+title: Licenses
+menuWeight: 1100
+enterprise: true
+---
+
+Mesosphere uses licenses to ensure that customers respect their contract terms. The DC/OS licensing component tracks the state of a cluster license, collects information to check if any licensing terms have been breached, and supports operations for updating the license when a contract is extended or changed.
+
+A cluster license file contains all of the information to determine if the terms of a contract agreement has been breached. It also contains the set of keys necessary to encrypt and decrypt audit data collected by the DC/OS licensing component. 
+
+
+# Configuring the cluster license
+
+You specify the license file as a base64 encoded string when you configure a cluster installation. The DC/OS licensing component will launch successfully only if the information in the license is legitimate. Once the DC/OS licensing component launches, the rest of the DC/OS components are started.
+
+To configure a cluster license, specify the license text received in email sent by your Authorized Support Contact in the [`dcos_license`](/1.11/installing/ent/custom/configuration/configuration-parameters/#dcos-license-enterprise) property of the `config.yaml` file. 
```

Comment:
Setting manually the config.yaml field is not necessary. Only a file at the same level as the detect_ip script is needed. (cc @branden)

Reply:
@jimenez  is there a dcos_license field in config.yaml? If so, what is the purpose of the property?  If a file is needed at the same level as detect_ip script, is this the place to discuss that? https://docs.mesosphere.com/1.10/installing/ent/custom/advanced/#configure-your-cluster

Reply:
There's no config.yaml field for the license, users will provide it as a file they create at `genconf/license.txt` before running the onprem installer, similar to the IP detect script. https://docs.mesosphere.com/1.10/installing/ent/custom/advanced/ looks like a good place to document it, however this is an enterprise-only feature, and that doc appears to cover both enterprise and open. I'm not sure what's the best way to organize that.

Reply:
That doc is enterprise only. It's in the `ent` folder.

Reply:
Oh yup. Yeah then I'd say that's a great place to cover it.

Reply:
@branden  If you need to run the advanced install procedure to specify a license, does that mean that CLI and GUI installer are not supported for enterprise customers?

Reply:
The CLI installer is supported, you provide it the license key under `genconf/` in the same way as the advanced install. For enterprise onprem installations, we should document the license key file in the same way that we document the ip-detect script, since they're used similarly. Both are files you are required to write to a specific location before running an advanced or CLI install.

The GUI installer is unsupported for enterprise customers until we update it to support this new required config input. /cc @klueska.